### ISSUE TITLE
Optimize variable size calculation in Li3perf.php

### ISSUE THREAD

**Issue by mariano** [tmaiaroto/li3_perf#9]
So I was looking at the `Li3perf::varsize()` method. I totally get how doing the actual memory calculation of a copied variable seems to be the only real, practical way of getting the variable size.

However, wouldn't it be much simpler if instead of manually copying the variable, do an unserialize & serialize? Like so:

``` php
$current_mem = memory_get_usage();
$tmp_var = unserialize(serialize($var));
$variable_size = memory_get_usage() - $current_mem;
unset($tmp_var);
```

I'm asking because you may have already looked into this approach. I've quickly tested this and it produces the same output as the `_varCopy()` approach.

If you are comfortable with this change, I can go ahead and make a pull request.

**Comment by mariano** [comment]
Actually on second thought the serialization of a closure is impossible, which would conflict with any variable that contains a  closure. So nevermind this, closing it.

**Comment by tmaiaroto** [comment]
I have tried serialize() and unserialize(). It's more inaccurate from what I've read. For certain types of variables. There were about 3 or 4 different approaches that I found while researching it and this was the one that made the most sense to me...But I'm open to changing it if there's some way to prove accuracy. I just seemed to get numbers that were all over the place trying different methods. 

Another method I saw was to store the variable in APC and see what it took there. The only problem is, even a null value seemed to take 512 bytes. It must reserve a certain amount of space or also include some sort of consistent overhead.
